src/libraries/Common/src/Interop/Windows/WinSock/Interop.TransmitFile.cs

```diff
@@ -1,7 +1,7 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
-
+#nullable enable
```

Comment:
We normally empty line in between license header and `#nullable enable`

Reply:
Fixed all of them.